packages/discovery-provider/src/api/v1/helpers.py

```diff
@@ -816,7 +816,7 @@ def __schema__(self):
 )
 
 full_search_parser = pagination_with_current_user_parser.copy()
-full_search_parser.add_argument("query", required=True, description="The search query")
+full_search_parser.add_argument("query", required=False, description="The search query")
```

Comment:
did we want this to go in now?

Reply:
I think yeah it's fine for this to go now